libs/kml/serdes_geojson.cpp

```diff
@@ -0,0 +1,224 @@
+#include "kml/serdes_geojson.hpp"
+#include "kml/color_parser.hpp"
+
+#include "geometry/mercator.hpp"
+
+#include <map>
+#include <string>
+
+namespace kml
+{
+namespace geojson
+{
+
+std::string DebugPrint(GeoJsonGeometry const & g)
+{
+  if (std::holds_alternative<GeoJsonGeometryPoint>(g))
+  {
+    auto geoPoint = std::get_if<GeoJsonGeometryPoint>(&g);
+    return DebugPrint(*geoPoint);
+  }
+  else if (std::holds_alternative<GeoJsonGeometryLine>(g))
+  {
+    auto geoLine = std::get_if<GeoJsonGeometryLine>(&g);
+    return DebugPrint(*geoLine);
+  }
+  else
+  {
+    auto geoUnknown = std::get_if<GeoJsonGeometryUnknown>(&g);
+    return "GeoJsonGeometryUnknown [type = " + geoUnknown->type + "]";
+  }
+}
+
+std::string DebugPrint(JsonTMap const & p)
+{
+  std::ostringstream out;
+  bool isFirst = true;
+  out << "{";
+  for (auto const & pair : p)
+  {
+    // Add seperator if needed
+    if (isFirst)
+      isFirst = false;
+    else
+      out << ", ";
+
+    out << '"' << pair.first << "\" = " << DebugPrint(pair.second) << ", ";
+  }
+  return out.str();
+}
+
+std::string DebugPrint(glz::json_t const & json)
+{
+  std::string buffer{};
+  if (glz::write_json(json, buffer))
+    return buffer;
+  else
+    return "<JSON_ERROR>";
+}
+
+bool GeojsonParser::Parse(std::string_view json_content)
```

Comment:
nit: code style (not a Python)
```suggestion
bool GeojsonParser::Parse(std::string_view jsonContent)
```